lua/lspconfig/server_configurations/omnisharp.lua

```diff
@@ -51,32 +55,21 @@ return {
       table.insert(new_config.cmd, '--languageserver')
 
       -- Append configuration-dependent command arguments
-      if new_config.enable_editorconfig_support then
-        table.insert(new_config.cmd, 'FormattingOptions:EnableEditorConfigSupport=true')
+      local function flatten(tbl)
```

Comment:
why not use `vim.tbl_flatten()` ?

Reply:
I didn't know about such function, but after trying it, it returns an empty table, so it probably does something completely different from what we want here.

Reply:
I see now - that function works on lists, this is a normal table/map/dict.